docs/learn/migrate-to-lukso.md

```diff
@@ -6,15 +6,80 @@ description: What to consider when building on LUKSO with LSPs (LUKSO Standard P
-- The [Universal Profile Browser Extension](/install-up-browser-extension) automatically handles permissions checks when interacting with dApps. If needed, signatures can be verified manually by [retrieving the controllers](../learn/key-manager/get-controller-permissions.md) of an account and checking their permissions.
-- As developers interact with smart contracts with different or custom functionalities, verifying that certain conditions and methods are set before interacting with them is always recommended. Such checks can be done by [detecting interfaces and metadata](../learn/standard-detection.md) of the given address.
-- When doing network interactions without a [Transaction Relay Service](../standards/relayer-api.md), the [controller](./concepts.md#controller) of the Universal Profile has to be funded to execute network transactions on behalf of the account.
+For more information regarding the network and standards, see the [Introduction](./introduction.md) and [Concepts & Glossary](./concepts.md) pages.
 
-## Deploying Ethereum Contracts to LUKSO
+:::
 
-As LUKSO is EVM compatible, all smart contracts written for Ethereum can easily be ported over and redeployed. The main difference is that developers must connect to a [LUKSO node or RPC endpoint](../networks/mainnet/parameters) instead of an Ethereum node.
+## Migrating from other EVM Chain
+
+As LUKSO is an EVM-compatible chain (_"Ethereum twin"_), any smart contract written for Ethereum (compiling to EVM bytecode) can be re-deployed on LUKSO. You only need to set up your application to connect to a [LUKSO RPC endpoint](../networks/mainnet/parameters) to deploy it on the LUKSO chain.
+
+Developers are encouraged to use the LSP standards to leverage their features and improve the features and overall experience of their protocols. Therefore, some codebases might require to be adjusted to convert ERC interactions to LSP interactions. The section below provides more details.
+
+## From Wallets → to 🆙
+
+<div style={{textAlign: 'center', color: 'grey'}}>
+  <img
+    src={require('../../static/img/learn/profile-metadata-same-across-dapps.png').default}
+    alt="`Examples of interacting with contracts, like minting tokens or refining burntpix."
+    width="1200"
+  />
+<br/>
+<i>3 x different dApps (UniversalProfile.cloud, Universal.Page and UniversalSwaps.io), that same Universal Profile's information shared across each dApp.</i>
+<br /><br />
+</div>
+
+Any traditional web3 wallet (Metamask, etc.) can interact with dApps on LUKSO. [Add LUKSO the list of networks](../networks/testnet/parameters.md) in your wallet and switch to the LUKSO chain. You can start holding assets (LYX, tokens, and NFTs), transfer them, and deploy and interact with smart contracts.
```

Comment:
More concise version:

Any traditional web3 wallet (e.g., Metamask) can interact with LUKSO dApps. Simply [add LUKSO to your wallet](https://github.com/lukso-network/docs/pull/networks/testnet/parameters.md) and switch to the LUKSO chain.